proto/stat_var.proto

```diff
@@ -239,8 +230,6 @@ message SearchStatVarResponse {
   // A ranked list of stat vars that aren't included in any of the returned stat
   // var groups.
   repeated EntityInfo stat_vars = 1;
-  // A list of stat var group results ranked by relevance.
```

Comment:
can we still mark this field as deprecated, just in case custom DC got proto field error.

Reply:
+1 `reserved 2;  // Deprecated "stat_var_groups"`

Reply:
done